executor/oci/user.go

```diff
@@ -91,6 +91,7 @@ func parseUID(str string) (uint32, error) {
 // once the PR in containerd is merged we should remove this function.
```

Comment:
```
// WithUIDGID allows the UID and GID for the Process to be set
// FIXME: This is a temporeray fix for the missing supplementary GIDs from containerd
// once the PR in containerd is merged we should remove this function.
```

Have these changes been merged in containerd, and if so, can we remove this function (making it an alias for the one in containerd? https://github.com/containerd/containerd/blob/8cb00f45c973b017f403d05d4d208ab2971ff335/oci/spec_opts.go#L692-L702

Reply:
The function arguments are different, so it still can't be removed